qemu/tests/qemu_guest_agent.py

```diff
@@ -813,29 +813,43 @@ def _action_before_fsfreeze(self, *args):
 
     @error_context.context_aware
     def _action_after_fsfreeze(self, *args):
-        error_context.context("Verfiy FS is frozen in guest.")
+        error_context.context("Verify FS is frozen in guest.", logging.info)
         if not isinstance(args, tuple):
             return
 
         if not self._open_session_list:
-            raise error.TestError("Could not find any opened session")
+            self.test.error("Could not find any opened session")
         # Use the last opened session to send cmd.
         session = self._open_session_list[-1]
         try:
-            session.cmd(self.params["gagent_fs_test_cmd"])
+            # The frozen state is limited for up to 10 seconds by VSS.
+            session.cmd(self.params["gagent_fs_test_cmd"], timeout=9)
         except aexpect.ShellTimeoutError:
-            logging.debug("FS freeze successfully.")
+            logging.info("FS is frozen,can't write in guest.")
         else:
-            raise error.TestFail("FS freeze failed, guest still can"
-                                 " write file")
+            self.test.fail("FS is not frozen,still can write in guest.")
 
     @error_context.context_aware
     def _action_before_fsthaw(self, *args):
         pass
 
     @error_context.context_aware
     def _action_after_fsthaw(self, *args):
-        pass
+        error_context.context("Verify FS is thawed in guest.", logging.info)
+        if not isinstance(args, tuple):
```

Comment:
*args makes args always be a tuple.
Do you have any cases make it returns?

Reply:
yes, you are right. 
Modified, pls check it again,thanks.